ci/requirements-2.7.run

```diff
@@ -19,3 +19,4 @@ pymysql=0.6.3
 html5lib=1.0b2
 beautiful-soup=4.2.1
 statsmodels
```

Comment:
I think you have add ``py`` here as well (otherwise these will skip). Note on the 2.6 build these SHOULD skip.